tilequeue/store.py

```diff
@@ -59,6 +59,29 @@ def parse_coordinate_from_path(path, extension, layer):
                         pass
 
 
+def _backoff_and_retry(num_tries=5, factor=2, interval=1, logger=None):
+    from time import sleep
+
+    # do the first num_tries-1 attempts wrapped in something to catch any
+    # exceptions, optionally log them, and try again.
+    for try_counter in xrange(1, num_tries):
+        try:
+            yield
+            break
+
+        except Exception as e:
```

Comment:
Interesting idea to use a generator, but I don't think that it'll actually catch the exception here and retry.